packages/backend/jest.setup.js

```diff
@@ -7,6 +7,8 @@ const POSTGRES_PORT = 5432
 const REDIS_PORT = 6379
 
 module.exports = async () => {
+  const workers = parseInt(process.env.JEST_WORKERS || '1')
```

Comment:
What happens if the number of actual workers exceeds `process.env.JEST_WORKERS`?
It'd be nice if this jest had a way for the global setup file to know the number of workers, but I'm not seeing one.

Reply:
[`--max-workers`](https://jestjs.io/docs/26.x/cli#--maxworkersnumstring)
>this defaults to the number of the cores available on your machine minus one for the main thread.

Could we ditch `process.env.JEST_WORKERS ` and use that default value?

```suggestion
  const workers = os.cpus().length - 1
```

Reply:
https://github.com/facebook/jest/issues/8708

I guess we could just write a script that counts how many test files there are rather.

Reply:
> Could we ditch process.env.JEST_WORKERS and calculate that default value ourselves?

Didn't see that, that could work :+1:

Reply:
Actually, `maxWorkers` is accessible in [globalSetup](https://jestjs.io/docs/26.x/configuration#globalsetup-string):
>This function gets Jest's `globalConfig` object as a parameter.

So no computation required

Reply:
> This function gets Jest's globalConfig object as a parameter.

That's not actually accessible from the function, that's just for storing global values like db configs that need to be accessed in teardown.